app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by(id: params[:id])
```

Comment:
bỏ ngoặc

đã nhắc thêm code xử lý trường hợp user bị nil nè, lì gớm, thêm vào hì

Reply:
find & find_by!: raise exception. find_by is not.